components/base/agilex/limo_base.go

```diff
@@ -162,7 +160,7 @@ func CreateLimoBase(ctx context.Context, conf resource.Config, logger golog.Logg
 	}
 	globalMu.Unlock()
 
-	base := &limoBase{
+	lb := &limoBase{
```

Comment:
nit: in the future, please have cleanup and variable renaming to separate PRs to make it easier to review the salient changes.

Reply:
couldn't do that here because I needed to get `base.Features` for get properties, and the created limobase was called `base`.

We shouldn't call any component struct the same as the component package name in the future.